com.unity.render-pipelines.universal/Shaders/Unlit.shader

```diff
@@ -234,7 +234,7 @@ Shader "Universal Render Pipeline/Unlit"
 #endif
 
                 color = MixFog(color, input.fogCoord);
-                alpha = OutputAlpha(alpha);
+                alpha = OutputAlpha(alpha, _Surface);
```

Comment:
This fixes an issue where unlit objects were being rendered as opaque.

Reply:
I know _Surface isn't originally from this PR, but I think it's pretty bad name. I'm like surface of what? What aspect of surface? What is this? every time I see it.
I mean most of the material parameters are part of object surface/material description in some way.
Could we change that before we release?

Reply:
Yes, we wanted to have `_SurfaceType` but that requires another material upgrader which makes things complicated. `_Surface` is already in our shaders so we simply used the data that was already present. 

Not in the scope of this PR.